Solutions/PaloAltoCDL/Hunting Queries/PaloAltoCDLOutdatedAgentVersions.yaml

```diff
@@ -0,0 +1,27 @@
+id: 555bf415-e171-4ad2-920f-1a4a96a9644c
+name: PaloAlto - Agent vesions
+description: |
+  'Query shows agent versions'

```

Comment:
Should the description say something about showing agents that are not on the latest version as we are using | where PanOSAgentVersion != cur_ver in the query.